tests/testthat/example/out.R

```diff
@@ -130,7 +130,7 @@ test <- function() {
   difficult(nested(
     "function", call
   ),
- with, more, args
+    with, more, args
```

Comment:
Same here, this is flat styling. Maybe we should remove tests for it anyways when implementing #121.